File: Level 7kyu/Disemvowel Trolls.js
function disemvowel(str) {
    const includesArray = ['u', 'e', 'o', 'a', 'i', 'U', 'E', 'O', 'A', 'I'];
    for (let i = 0; i < str.length; i++) {
        if (includesArray.includes(str[i])) {
            str = str.replace(str[i--], '');
        }
    }
    return str;
}

File: Level 7kyu/Disemvowel Trolls.rs
// Submitted
fn disemvowel(s: &str) -> String {
    let str = s.replace(&['A','E','I','O','U','a','e','i','o','u',][..], "");
    str
}
  
// Run to test
fn main() {
    let res = disemvowel("huynh viet tuan kiet");
    println!("{}", res);
}
